tests/beaker_tests/cisco_bgp_neighbor/test_bgpneighbor.rb

```diff
@@ -80,6 +80,7 @@
     timers_keepalive:       '90',
     timers_holdtime:        '270',
     update_source:          'loopback151',
+    peer_type:              'fabric-external',
```

Comment:
Should you also test for `'default'` and `'fabric-border-leaf'`?

Reply:
`default` is not supported, but, I've added tests for `fabric-border-leaf`

Reply:
added tests for default now too